src/components/PageTransition/index.jsx

```diff
@@ -0,0 +1,33 @@
+import React, { Component } from 'react';
+import MatButton from '@material-ui/core/Button';
```

Comment:
We have a `Button` component that is basically a wrapper around `@material-ui/core/Button`. `src/components/Button`. Also, rename `MatButton` to `Button`.